spec/workers/zendesk_ticket_worker_spec.rb

```diff
@@ -18,4 +18,15 @@
 
     expect(a_request(:post, %r{.*/tickets/.*})).to_not have_been_made
   end
+
+  it "discards the ticket if it receives a 409 response" do
+    zendesk_has_user(email: "a@b.com", "suspended" => false)
```

Comment:
The reason was being consistent with other tests on this page. But I've now amended them too in 74d59133.